src/tests/calculate.test.js

```diff
@@ -0,0 +1,24 @@
+import calculate from '../components/logic/calculate';
```

Comment:
- [ ] There ought to be the usage of Jest snapshots for testing all the components i.e files in the components and pages folders. Please add them.

Reply:
Well Addressed !!!